stdlib/public/core/LifetimeManager.swift

```diff
@@ -283,3 +283,40 @@ extension String {
 public func _copy<T>(_ value: T) -> T {
   copy value
 }
+
+/// Unsafely discard any lifetime dependency on the `dependent` argument. Return
+/// a value identical to `dependent` with a lifetime dependency on the caller's
+/// borrow scope of the `borrows` argument.
+///
+/// TODO: Remove @_unsafeNonescapableResult. Instead, the unsafe dependence
+/// should be expressed by a builtin that is hidden within the function body.
+@_unsafeNonescapableResult
+@_alwaysEmitIntoClient
+@inline(__always)
+@lifetime(borrow source)
+internal func _unsafeLifetime<T: ~Copyable & ~Escapable,
+                              U: ~Copyable & ~Escapable>(
+  dependent: consuming T, borrows source: borrowing U
+) -> T {
```

Comment:
The stdlib uses a [very particular indentation style](https://github.com/swiftlang/swift/blob/main/docs/StandardLibraryProgrammersManual.md#line-breaking
); please do not deviate from it.

The stdlib (and adjacent codebases) never align continuation lines on arbitrary columns like the declaration of `U` above.

> Our primary rule is that if we need to insert a line break anywhere in the middle of a list (such as arguments, tuple or array/dictionary literals, generic type parameters, etc.), then we must go all the way and put each item on its own line, indented by +1 unit, even if some of the items would fit on a single line together.
> [...]
> For lists that have delimiter characters ((/), [/], </>, etc.), we prefer to put a line break both after the opening delimiter, and before the closing delimiter. However, within function bodies, it's okay to omit the line break before the closing delimiter.

```suggestion
internal func _unsafeLifetime<
  T: ~Copyable & ~Escapable,
  U: ~Copyable & ~Escapable
>(
  dependent: consuming T, borrows source: borrowing U
) -> T {
```